src/renderer/components/appveyor.yml

```diff
@@ -0,0 +1,20 @@
+environment:
+  matrix:
+    - nodejs_version: "8"
+
+init:
+  - git config --global core.symlinks true
+
+install:
+  - ps: Install-Product node $env:nodejs_version x64
+  - node --version
+  - npm i
```

Comment:
Perhaps use latest NPM + [`npm ci`](https://blog.npmjs.org/post/171556855892/introducing-npm-ci-for-faster-more-reliable)?